util/helpers.js

```diff
@@ -46,7 +47,16 @@ function mergeRules (api, opt, ext) {
 
   addImports(api, 'lists', data, end)
 
-  opt.data = data.join('\n')
+  let sassLoaderVersion
+  try {
+    sassLoaderVersion = semver.major(require('sass-loader/package.json').version)
```

Comment:
`sass-loader` should be declared in `peerDependencies`

Reply:
Why?

Reply:
To support Yarn PnP, but most importantly, to ensure the package manager is aware of it so it can place it in a location where `vue-cli-plugin-vuetify` can require it.

/ cc @arcanis